### DISCUSSION TITLE
用了一天，个人的问题和建议。自定义节点后的问题，加了一些shape，就拖拽不了了，选中样式也不好加

### DISCUSSION THREAD

**Discussion by Mark123654** [antvis/G6#5407] [Q&A] [answered]
1、自定义节点后的问题，加了一些shape，就拖拽不了了，选中样式也不好加
2、官网的钉钉群都是失效了。有没有其他交流的地方，方便大家交流学习，自己摸索，是在是太麻烦了
3、内置节点配置太少，要实现的样式又复杂，所以要用自定义，但是自定义，看官网用不明白，唉
4、建议多加一点符合实际场景的自定义节点实例，如果单纯用官方实例那到没啥难度，但是（在复杂场景）上手感觉有点难度，

**Comment by hustcc** [comment, marked as answer]
G6 5.0 吗？

G6 5.0 目前还在做一些内部改造，反馈收到了，会进行改造。一些技术设计文档在[这里](https://www.yuque.com/antv/g6/iwmdk3u32uwrsmi5)，有兴趣的话，也可以一起进来写代码，从简单的入手来。

**Reply by Mark123654** [reply]
我现在用的是4.8，再建几个交流群把，多开几个，方便大家沟通交流，谢谢😘

**Reply by hustcc** [reply]
看 readme 中有两个交流群。
